---
string camera_name
sensor_msgs/CameraInfo camera_info
